Dockerfile.common.in

```diff
@@ -1,26 +1,20 @@
 FROM ${BASE_IMAGE_NAME}:${PHP_VERSION}
 
-LABEL authors="${AUTHOR}"
-
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 USER root
 
+RUN echo 'deb http://ftp.us.debian.org/debian sid main' | tee -a /etc/apt/sources.list
+# hadolint ignore=DL3009
+RUN apt-get update
 # hadolint ignore=DL3008
-RUN echo 'deb http://ftp.us.debian.org/debian sid main' | tee -a /etc/apt/sources.list && \
-    curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
-    apt-get install --no-install-recommends -y \
-      build-essential \
-      mysql-client-5.7 \
-      nodejs \
-      subversion \
-      && \
-    rm -rf /var/lib/apt/lists/* && \
-    docker-php-ext-install mysqli
-
-RUN npm install -g \
-      stylelint@${STYLELINT_VERSION} \
-      eslint@${ESLINT_VERSION}
+RUN apt-get install --no-install-recommends gcc-8-base -y
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+# hadolint ignore=DL3008
+RUN apt-get install --no-install-recommends -y build-essential mariadb-client nodejs npm subversion
+RUN rm -rf /var/lib/apt/lists/*
+RUN docker-php-ext-install mysqli
+RUN npm install -g stylelint@${STYLELINT_VERSION} eslint@${ESLINT_VERSION}
```

Comment:
Not necessarily to change in this PR, but it would make more sense to have a separate job (and image) for PHP, and one for CSS+JS. It would make each image a lot simpler too (less layers), and probably faster, though it's already a lot faster than the concurrent other jobs.

Reply:
You made me go through the rabbit hole of trying to see how easy that separation would be 😛 
So I added 2 more commits in this PR. We now have `stylelint` and `eslint` in a different docker image.

One thing (probably for a future PR) is to re-enable stylelint and eslint bats tests. By default bats tests run on php containers, so these two were failing.

Reply:
Nice one :100: Yeah the bats tests don't seem that important for this image, as it doesn't need to handle any sensitive data, we can skip them for now.